src/Runner/Job.php

```diff
@@ -245,4 +245,24 @@ public function getHeaders()
 		return $this->headers;
 	}
 
+
+	/**
+	 * @return string
+	 */
+	private function buildArgumentsList()
+	{
+		$args = [];
+
+		foreach ($this->args as $key => $arg) {
```

Comment:
Potential BC break: Job's arguments are now held in an associative array to provide more semantics. This implementation effectively disallows multi-valued options unless they are passed in the current fashion. Could this be a problem?